### PR TITLE
TAJO-1393: Resolve findbug warnings on Tajo Cli Module

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
Pass the following two steps.

* cd tajo-cli; mvn compile findbugs:findbugs
* mvn clean install -Pdist -Dtar -Dhadoop.version2.6.0

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
Hi, Travis CI build is done~

File: tajo-cli/src/main/java/org/apache/tajo/cli/tools/TajoGetConf.java [hunk 1]
```diff
@@ -119,6 +119,9 @@ public void runCommand(String[] args, boolean tsqlMode) throws Exception {
     } else if (hostName == null && port == null) {
       tajoClient = new TajoClientImpl(ServiceTrackerFactory.get(tajoConf));
     }
+    if (!tajoClient.isConnected()) {
```
Comment (hyunsik) [inline tajo-cli/src/main/java/org/apache/tajo/cli/tools/TajoGetConf.java]: Actually, this class does not need TajoClient. I think that removing tajoClient is a better solution. So, you don't need the below ``System.out.print`` statement.

File: tajo-cli/src/main/java/org/apache/tajo/cli/tools/TajoHAAdmin.java [hunk 2]
```diff
@@ -131,6 +131,9 @@ public void runCommand(String[] args) throws Exception {
     } else if (hostName == null && port == null) {
       tajoClient = new TajoClientImpl(ServiceTrackerFactory.get(tajoConf));
     }
+    if (!tajoClient.isConnected()) {
```
Comment (hyunsik) [inline tajo-cli/src/main/java/org/apache/tajo/cli/tools/TajoHAAdmin.java]: I think that this class also does not require TajoClient connection. Removing TajoClient would be a better solution to fix 'unused field' warning.

**Comment by hyunsik** [comment]
I leave some comments. TajoHAAdmin seems to need refactoring. Its code is somewhat messy.

**Comment by dongjoon-hyun** [comment]
Thank you. I will rebase and remove them.

**Comment by dongjoon-hyun** [comment]
I've done and the followings are passed.
cd tajo-cli; mvn compile findbugs:findbugs
mvn clean install -Pparallel-test,hcatalog-0.12.0 -DLOG_LEVEL=ERROR -Dmaven.fork.count=2

**Comment by hyunsik** [comment]
The patch looks good to me. I'll finish the review when CI is completed.

**Comment by dongjoon-hyun** [comment]
Thanks, @hyunsik

**Pull request closed without merging**